dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -197,7 +199,14 @@ static MethodSymbolReference constantToMethodReference(ConstantMethodref constan
               + nameAndTypeIndex
               + ". This class file is not compliant with CONSTANT_Methodref_info specification");
```

Comment:
Now we don't know whether it's a method or field or what, so can this message be more general?

Reply:
Updated.